Command/AlignCommand.php

```diff
@@ -55,6 +55,31 @@ protected function configure()
                 InputArgument::REQUIRED,
                 'Environment target name'
             )
+            ->addArgument(
+                'scrollSize',
+                InputArgument::OPTIONAL,
+                'Size of the elasticsearch scroll request',
+                100
+            )
+            ->addArgument(
+                'scrollTimeout',
+                InputArgument::OPTIONAL,
+                'Time to migrate "scrollSize" items i.e. 30s or 2m',
+                '1m'
+            )
+            ->addArgument(
+                'contentType',
+                InputArgument::OPTIONAL,
+                'The content type you wish to align',
+                '1m'
+            )
+            ->addOption(
+                'searchQuery',
```

Comment:
In what situation are we going to be doing this ?

Reply:
Mathieu wanted this, perhaps if we need to filter further. For example only documents that were published before a certain date.